kbs/src/attestation/backend.rs

```diff
@@ -230,6 +231,8 @@ impl AttestationService {
         attestation: &[u8],
         request: HttpRequest,
     ) -> anyhow::Result<HttpResponse> {
+        ATTESTATION_REQUESTS.inc();
```

Comment:
nit: Remote attestation (RCAR protocol) consists of two rounds of API calls, `auth` is the first round and `attest` is the second round. I'm not sure if this is appropriate here, maybe we can add some context like [here](https://github.com/confidential-containers/trustee/blob/main/kbs/src/plugins/implementations/resource/backend.rs#L129)?

Reply:
That makes it a lot clearer, thanks!  Wouldn't `auth` be best tracked then if we add a separate set of counters for it? If I understand correctly `auth` as such can't really fail (unless there is a request deserialisation or nonce generation error, or a version mismatch, it will always respond with a session cookie and thus succeed), `auth_requests_total` and `auth_requests_errors` whould seem enough.